scout/server/blueprints/variants/forms.py

```diff
@@ -105,14 +105,17 @@ class VariantFiltersForm(FlaskForm):
     load_filter = SubmitField(label="Load filter")
     delete_filter = SubmitField(label="Delete filter")
 
-    chrom = SelectField("Chromosome", [validators.Optional()], choices=CHROMOSOME_OPTIONS)
+    chrom = SelectField(
+        "Chromosome", [validators.Optional()], choices=CHROMOSOME_OPTIONS, default=""
+    )
     start = IntegerField("Start position", [validators.Optional()])
     end = IntegerField("End position", [validators.Optional()])
     cytoband_start = NonValidatingSelectField("Cytoband start", choices=[])
     cytoband_end = NonValidatingSelectField("Cytoband end", choices=[])
 
     filter_variants = SubmitField(label="Filter variants")
     export = SubmitField(label="Filter and export")
+    clinical_filter = SubmitField(label="Clinical filter")
```

Comment:
Right, but this isn't really supposed to be instantiated, and Cancer does not have one (yet)?

Reply:
This is not really affecting the query I guess, shall I revert this code?

Reply:
I think it would be less confusing, as long as Cancer does not have one,  or what do you think?

Reply:
I can remove it!

Reply:
Done!